plugins/jfrog-artifactory/src/components/JfrogArtifactoryRepository/tableHeading.tsx

```diff
@@ -4,19 +4,24 @@ import makeStyles from '@material-ui/core/styles/makeStyles';
 
 export const columns: TableColumn[] = [
   {
-    title: 'Version',
-    field: 'name',
+    title: 'Repos',
```

Comment:
I don't think "Repos" is the correct column header to display here. We should strive for accuracy and proper English here. Does this:

 ```suggestion
    title: 'Repository',
```

makes sense here?

Reply:
Agreed! Should be `Repository`